cloud/jenkins/pxc_operator_aks_latest.groovy

```diff
@@ -1,75 +1,99 @@
+aksLocation='westeurope'
 tests=[]
 clusters=[]
 
-void createCluster(String CLUSTER_SUFFIX) {
-    clusters.add("${CLUSTER_SUFFIX}")
+void prepareNode() {
```

Comment:
This file is missing `stash`/`unstash`?

Reply:
When we don't run tests in parallel clusters, we don't need stash/unstash. Previously, stash/unstash was used between checking out the sources and building the docker image. It was discovered that those stash/unstash was unnecessary.